wgengine/tstun/faketun.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/tailscale/wireguard-go/tun"
 )
 
-type fakeTUN struct {
+type FakeTUN struct {
```

Comment:
Add a comment now it's exported.

Reply:
I've actually been hoping we could not compile this at all in iOS prod builds to remove it from the binary size.

Looks like it's now wired in more.

We we not look at its concrete type and look for optional methods on it instead? Like `IsFakeTUN()`